src/main/java/com/adyen/service/LegalEntityManagement/HostedOnboardingPage.java

```diff
@@ -0,0 +1,40 @@
+package com.adyen.service.LegalEntityManagement;
+
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.constants.ApiConstants;
+import com.adyen.model.legalEntityManagement.JSON;
+import com.adyen.model.legalEntityManagement.OnboardingLink;
+import com.adyen.model.legalEntityManagement.OnboardingLinkInfo;
+import com.adyen.model.legalEntityManagement.OnboardingTheme;
+import com.adyen.model.legalEntityManagement.OnboardingThemes;
+import com.adyen.service.exception.ApiException;
+import com.adyen.service.resource.LegalEntityManagementResource;
+
+import java.io.IOException;
+
+public class HostedOnboardingPage extends Service {
+    public HostedOnboardingPage(Client client) {
+        super(client);
+        new JSON();
+    }
+
+    public OnboardingLink create(String legalEntityId, OnboardingLinkInfo request) throws IOException, ApiException {
+        String jsonRequest = request.toJson();
+        LegalEntityManagementResource resource = new LegalEntityManagementResource(this, String.format("/documents/%s", legalEntityId));
```

Comment:
Incorrect URL, this method should `POST` to [/legalEntities/{id}/onboardingLinks](https://docs.adyen.com/api-explorer/legalentity/2/post/legalEntities/_id_/onboardingLinks).

Reply:
Actually this one does POST! The ```resource.request()``` has a couple of overides in the Resoure.java class as Wouter let me know. The instance where you only fill in a Json string it will post automatically.

Reply:
My bad misunderstood your comment :) changed it!